channel-sdk/src/main/java/com/zello/channel/sdk/SessionContextAndroid.kt

```diff
@@ -27,6 +29,7 @@ internal class SessionContextAndroid(context: Context) : SessionContext {
 	private var handler: Handler = Handler()
 
 	override val transportFactory: TransportFactory = WebSocketsTransportFactory()
+	override val imageMessageManager: ImageMessageManager = ImageMessageManagerImpl()
```

Comment:
Wondering if this and `transportFactory` ought to be passed into the constructor? Maybe it would be overkill, since these are unlikely to ever change

Reply:
Customers will need to construct new `Session` objects, and I don't want them to need to know about the dependency injection internals. This way they just need to build a `SessionContext` with its public API and everything gets set up correctly.